### PR TITLE
[IMPROVEMENT] Lazy fetch server info

### PR DESCRIPTION
**Pull request by diegolmello**
@RocketChat/ReactNative

Check if user is logged to hide splash screen first and fetch server info later.
Also, stop alerting "Wrong url" in case of server is down.
The user is logged in, so the url is correct.

### CODE REVIEW

File: app/sagas/selectServer.js [hunk 1]
```diff
@@ -11,11 +11,13 @@ import database from '../lib/realm';
 import log from '../utils/log';
 import I18n from '../i18n';
 
-const getServerInfo = function* getServerInfo({ server }) {
+const getServerInfo = function* getServerInfo({ server, raiseError = true }) {
 	try {
 		const serverInfo = yield RocketChat.getServerInfo(server);
 		if (!serverInfo.success) {
-			Alert.alert(I18n.t('Oops'), I18n.t(serverInfo.message, serverInfo.messageOptions));
+			if (raiseError) {
```
Comment (IlarionHalushka) [inline app/sagas/selectServer.js]: why did you add `raiseError` parameter?
I suppose that checking `if (!serverInfo.success) {` would be enough to show error Alert.
And besides that `getServerInfo` is called only once in `handleSelectServer`: `getServerInfo({ server, raiseError: false });` so `raiseError` will always be falsy...

File: app/sagas/selectServer.js [hunk 2]
```diff
@@ -11,11 +11,13 @@ import database from '../lib/realm';
...
+				Alert.alert(I18n.t('Oops'), I18n.t(serverInfo.message, serverInfo.messageOptions));
```
Comment (IlarionHalushka) [inline app/sagas/selectServer.js]: maybe better use `showToast`?

File: app/sagas/selectServer.js (hunk 1 above)
Comment (diegolmello) [reply app/sagas/selectServer.js]: `getServerInfo` is called on `handleSelectServer`, `handleServerRequest` and `deepLinking`.
I don't want to raise the error every time.

File: app/sagas/selectServer.js (hunk 2 above)
Comment (diegolmello) [reply app/sagas/selectServer.js]: I want an action from the user.
